conda_gitenv/deploy.py

```diff
@@ -50,87 +49,61 @@ def deploy_tag(repo, tag_name, target, pkg_cache):
         raise ValueError("The tag '{}' doesn't have a manifested environment.".format(tag_name))
     with open(manifest_fname, 'r') as fh:
         manifest = sorted(line.strip().split('\t') for line in fh)
-    create_env(manifest, os.path.join(target, env_name, deployed_name), pkg_cache)
 
+    target = os.path.join(target, env_name, deployed_name)
+    create_env(repo, manifest, target, pkg_cache)
+
+
+def create_env(repo, pkgs, target, pkg_cache):
+    from conda.core.link import UnlinkLinkTransaction
+    from conda.core.package_cache import ProgressiveFetchExtract
+    from conda.exports import Resolve, fetch_index
+    from conda.models.channel import prioritize_channels
+    from conda.models.dist import Dist
+    from conda.gateways.disk.create import mkdir_p
+
+    spec_fname = os.path.join(repo.working_dir, 'env.spec')
+
+    # Skip branches that don't have an environment specification
```

Comment:
Don't really like this one. Is this new? Can we make this change separately and discuss?